### DISCUSSION TITLE
Training Asreview: what to do with 'maybe' articles?

### DISCUSSION THREAD

**Discussion by Hanedinges** [asreview/asreview#1451] [Q&A]
Dear all, 

Even with the best list of eligibility criteria, doubts can arise about the (ir)relevance of an article. What to do in such a situation? In Rayyan for example, it is possible to use the label 'maybe', add a reason for it and discuss it when all articles have been screened. However, AsReview doesn't have this feature. 

What to with 'maybe' articles? I see three options, but there might better ones of course;)

1. Including them...but this can disturb the learning process, right? Or is could there still be a reason to choose for this option? 
2. Discuss it __during__ this phase with a colleague instead of doing so after finishing the screening? 
3. Exclude it, add a note so it can be filtered. Continue screening after the stopping rule applies, discuss excluded articles with a note and add them manually to the included set. 

- Which option would be the best and why? Is there a guideline that I might have missed, a useful source?

- What are 'maybe' articles called in offical review-terms (so I can use it as a key word in my search for help)

- I'm also trying to think of the consequences of all options in terms of reliability of the review, if anyone is able to explain this it would be great!   

Thanks a lot!

Simone

**Comment by rohitgarud** [comment]
Hi @Hanedinges,
This is a well-known dilemma, check https://github.com/asreview/asreview/issues/328. 
I think the second option is the best one for now as this will not mess up the training by including or excluding in uncertain conditions. The decision will be binary and certain.
There is no official term for "maybe", because you have to make a decision at each record screening in the current version of ASReview.
I think this needs to be addressed once and for all @Rensvandeschoot @J535D165. We can add the "maybe" or "skip for now" option and bring back these "maybe" records at the end of the screening for reviewers to make the final decision. However, this can affect the performance of ASReview.

**Comment by FelixWdm** [comment]
Hi @Hanedinges , in addition to the useful comments of @rohitgarud: assuming you're going through multiple stages of screening, at some point these 'maybe's' need to be addressed. To not 'confuse' the algorithm and assuming the maybe-records are very similar in text to included ones, the logical option would be to include and add a comment. 
And regarding the terminology of 'maybe-records', since in the end you either include or exclude them, they will most likely be labelled as 'excluded during full text phase'. For reliability as well as reproducibility, each records must accounted for in each step. Ideally with the reasons for excluding per record as well.

**Comment by Rensvandeschoot** [comment]
Your query is indeed very insightful. In the context of active learning, making a definite decision on 'maybe' articles is crucial to optimizing the machine learning model's performance in ASReview.

While it can seem counterintuitive, the model actually learns the most from these 'borderline' or 'unclear' articles. The reason for this is that by making a clear decision (include/exclude) on these articles, you help refine the model's understanding of the nuances of what constitutes a relevant or irrelevant article for your specific review. This in turn helps improve the model's predictions for future articles, thereby enhancing the efficiency of your review process.

Therefore, if your goal is to maximally enhance the performance of ASReview, making a final decision on these articles is highly suggested.

**Comment by Hanedinges** [comment]
Sorry for my late reply guys! Your advice and explanation of the logic behind them are highly appreciated, thanks! 
The suggestion to include and add a note seems most practical and logical option (we are in the phase of TI-AB screening so we will go for that one. 

@FelixWdm wrote:  Ideally with the reasons for excluding per record as well.
Will this be a new feature in the future?

**Reply by Rensvandeschoot** [reply]
No worries about the delayed response! I'm glad to hear you found our advice helpful.

In terms of documenting reasons for exclusion, you can leverage the "notes" field in your current setup. If you maintain consistency in your statements or use specific codes for exclusion, it will allow you to effectively find or sort the records in your exported dataset based on these criteria. This way, you can keep track of the exclusion reasons even without a dedicated feature for it.

**Reply by rohitgarud** [reply]
Hi.. I think [this feature request](https://github.com/asreview/asreview/issues/1391) can be useful in this case.. otherwise, users have to manually type the same reasons again and again in the notes.. if custom tags/labels can be created, they can be used with few clicks.. and records can be filtered or sorted according to these tags/labels..

**Reply by Rensvandeschoot** [reply]
yes, I fully agree!!

**Reply by Hanedinges** [reply]
Thanks Rens and Rohitgarud:). Rohitgarud formulates  exactly what I was thinking when I read the idea of having to type codes in notes ! I wonder however what is more efficient for now....the workflow me and my colleague have agreed upon or typing a code in notes, export the set en filter, I would have to try. Our workflow now: we use ASReview and Rayyan in tandem, as we did for the pilot because of the convenience of a) highlighting keywords, b) select a reason for exclusion, c) chat while screening in case of any doubts d) working in the same project and thus be able to compare our work without having to export and import. However, it appears to me very inefficient to use two software programs:), this workflow cannot be what ASReview is meant for I suppose, right;)? We have a close deadline and already started, so I can no longer experiment much at the moment unfortunately. I wish I'd had the time to reach out before we started, it's too much work to switch to 'use codes in notes' option and we like the benefits of Rayyan....

**Reply by Hanedinges** [reply]
Just to be sure I understand it correctly: what we could (have) do(ne) is:
1. screen article
2. write down a code in a note that corresponds with the reason for exclusion
3. finally export results including notes
4. merge screening results researcher A en researcher B and compare both screening decisions and reasons
5. identify and solve conflicts (decisions or reasons for exclusion) Would it be possible to let excel identify/filter conflicts?

**Reply by Rensvandeschoot** [reply]
Thank you again for sharing your insights with us and for the valuable feedback. To address your points:

a) ASReview supports keyword highlighting. You can find more information about this in [this discussion](https://github.com/asreview/asreview/discussions/1243).

b) For selecting reasons for exclusion, you can indeed utilize the notes feature in ASReview. Although we're looking to improve this functionality, it can currently be used to mark exclusion reasons as you described above. You can then export the dataset and filter it based on these notes.

c) Regarding communication during the screening process, we prioritize privacy and do not have built-in chat functionality. We recommend using external communication tools for team discussions.

d) To compare the work between different team members, you can compute the Inter-Rater Reliability (IRR) using an extension developed by our community. You can find more information on this under MISC in [this discussion](https://github.com/asreview/asreview/discussions/1140).

I understand that switching to a new workflow in the middle of a project can be daunting. Please feel free to reach out if you have more questions or need further assistance.

**Reply by Hanedinges** [reply]
Hey hello,
Thanks for your reply! It's really a pity that we had to start as quick as we did and also that there was some misunderstanding about how much the team I just joined, knows about ASReview. It would be great I think if novices could start with a complete video tutorial, but I might have looked over it (as I thought my team already had basic knowledge)? My colleague sent me a link to a tutorial that was very easy to understand: [: [https://www.youtube.com/watch?v=eg9DPwMUSTA&list=PLg0Wf9RlqvSDf9J9I2rirb5Ah1y8wXBd4&index=1](https://eur05.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.youtube.com%2Fwatch%3Fv%3Deg9DPwMUSTA%26list%3DPLg0Wf9RlqvSDf9J9I2rirb5Ah1y8wXBd4%26index%3D1&data=05%7C01%7Cs.m.hanegraaff%40saxion.nl%7C9343fb626bf44bc13a6508db6be55a38%7Ca77b0754fdc14a62972c8425ffbfcbd2%7C0%7C0%7C638222405030873902%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=Uv%2FfcSHZpbJBt%2BptPuaSTWKc09dO3LkodSup5h0%2FYgI%3D&reserved=0)] 
I am sharing it with the intention to help increase the (ease of) use of ASReview,as i'm very enthousastic about it!
It would be a pleasure to me to help improve the ease of use of ASReview and/or lower the burden for researchers with little or no technical expertise nor curiousity: as I aim to specialize in doing reviews and become a go-to for researchers, for me it's worth the time investment and I also enjoy learning about and contributing to everything that increase efficiency of work flow. 

For now we do will continue with Rayyan and ASReview (...) but I will probably assist another team in August and I look forward to applying all lessons learnt here:)

**Reply by Rensvandeschoot** [reply]
You might be interested in our YouTube channel: https://www.youtube.com/@ASReview :-) 

If you are interested in creating videos, feel free to make these and we can put these on your channel

**Comment by Rensvandeschoot** [comment]
A PR has been opened for adding the option to add tags to the review screen: https://github.com/asreview/asreview/pull/1527. This functionality can be used to label `maybe` articles via a tag and double check these records in a later stage.
